src/nti/site/zcml.py

```diff
@@ -33,6 +34,10 @@ def registerSiteMapping(_context, source_site_name, target_site_name):
     """
     Create and register a site mapping, as a utility under the `source_site_name`.
     """
+    # Store and register in lowercase. Lookups occur with lowercased site
```

Comment:
It would be good if the documentation for the directive mentioned this. Otherwise it's completely not apparent why you can't find a site mapping.

ZCML is generally written by hand. So maybe, to avoid the inherent surprise of not being able to find what you typed, the directive should raise an exception if the names aren't entered in lower case?

Reply:
I think you're right. Talking this over with @cutz, it's a lot for this library about how this utility may be looked up in other places. I don't think it makes sense to change how this is implemented here.